cryodrgn/protocols/protocol_preprocess.py

```diff
@@ -146,8 +146,8 @@ def createOutputStep(self):
                                    ctfs=self._getFileName('output_ctfs'),
                                    dim=self._getBoxSize() + 1,
                                    samplingRate=self._getSamplingRate())
-        output.ptcls.set(self)
-        output.ptcls.setExtended("inputParticles")
+
+        output.ptcls= self.inputParticles.clone()
```

Comment:
Easiest way to get an "indirect pointer" is to clone() input params that are always indirect pointers (unless created form tests that are direct)